### ISSUE TITLE
cannot change backdrop

### ISSUE THREAD

**Issue by pfr-dev** [damnitharshit/nightly#7]
Not sure why, seems simple enough. 

[![Screenshot-from-2022-06-26-13-00-25.png](https://i.postimg.cc/2SfxRWmW/Screenshot-from-2022-06-26-13-00-25.png)](https://postimg.cc/vc3n5c48)

**Comment by rakaw** [comment]
yes, i am having the same problem. Also with changing icons as well.

**Comment by damnitharshit** [comment]
@pfr-dev could you please provide me with the following info:
- What browser are you? (and what version?)
- Did this happen after you changed something, or was this behaviour taking place by default? If you changed something, what did you change?

**Comment by pfr-dev** [comment]
Firefox 101.0.1 for NixOS

**Comment by rakaw** [comment]
@damnitharshit I'm on the same Firefox as pfr-dev on Arch Linux and the backdrop-filter property is enabled, but it still doesnt import pictures. I can only use online links for wallpaper and icons.

**Comment by pfr-dev** [comment]
> @pfr-dev have you enabled the backdrop-filter property in firefox?

Yes, I have this enabled. As @rakaw said, it works fine with online url's pointing to images, just not local files.

**Comment by damnitharshit** [comment]
Okay, Thanks for the info and sorry for the delay, but I'm working on fixing it.
